### PR TITLE
MM-13166 Fix image dimensions for singleImage attachment

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Fixes image dimensions for loading placeholder for images with height and width greater than the max dimensions available
#### Ticket Link
[MM-13166](https://mattermost.atlassian.net/browse/MM-13166)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Has UI changes

### CODE REVIEW

File: components/single_image_view/single_image_view.test.jsx [hunk 1]
```diff
@@ -100,35 +100,6 @@ describe('components/SingleImageView', () => {
         expect(wrapper.state('showPreviewModal')).toEqual(false);
     });
 
-    test('should match dimensions on computeImageDimensions', () => {
```
Comment (sudheerDev) [inline components/single_image_view/single_image_view.test.jsx]: Using a utils functions which is used in most places now. It covers more corner cases and is tested there for this scenario
